clients/privacy-center/cypress/e2e/consent-banner-gpp.cy.ts

```diff
@@ -57,6 +58,24 @@ describe("Fides-js GPP extension", () => {
     }).as("patchNoticesServed");
   });
 
+  describe("with GPP forced", () => {
```

Comment:
moved up from below just because it's more "global" and felt like it belonged here with the other "global" tests.